extensions/functions_arithmetic_decimal.yaml

```diff
@@ -174,3 +174,95 @@ aggregate_functions:
         decomposable: MANY
         intermediate: "DECIMAL<38,S>"
         return: "DECIMAL<38,S>"
+
```

Comment:
no need for a blank line between function definitions

Reply:
fixed it